ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -58,9 +59,9 @@ protocolInfoByron (NumCoreNodes numCoreNodes) (CoreNodeId nid) params gc =
                   --TODO: also allow not being a BFT leader:
```

Comment:
You can fix this TODO now if you make the secret key material optional.

Reply:
Done!